exercises/binary-search-tree/binary_search_tree_test.exs

```diff
@@ -0,0 +1,107 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("binary_search_tree.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule BinarySearchTreeTest do
+  use ExUnit.Case
+
+  test "retains data" do
+    assert BinarySearchTree.new(4).data == 4
+  end
+
+  @tag :pending
+  test "inserting lower number" do
+    four =
```

Comment:
Maybe instead of calling this `four` we can call it `root` or `root_node` or something like that? I think it might get a little confusing for some folks seeing `four.data == 4`.

Reply:
I agree and updated to 'root'. I kinda cargo culted the naming pattern here from other impls, but we can do better for the elixir repo :-)